packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -159,4 +171,11 @@ class TapGestureRecognizer extends PrimaryPointerGestureRecognizer {
 
   @override
   String get debugDescription => 'tap';
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder description) {
+    super.debugFillProperties(description);
+    description.add(new FlagProperty('wonArena', value: _wonArenaForPrimaryPointer, ifTrue: 'wonArena'));
```

Comment:
probably best to say `wonArenaForPrimaryPointer` as the name, but the "ifTrue" can just be "won arena" and you can set `showName` to false (otherwise it'll say something like "wonArena: wonArena" which is confusing)

Reply:
oh, never mind, i see apparently FlagProperty defaults the show name thing to false.

Reply:
Sorry the "never mind" was about the showName thing. Still best to say `wonArenaForPrimaryPointer` as the name. :-)

Reply:
for the ifTrue, let's just have `won arena` rather than camelCase